### PR TITLE
feat: export

### PR DESCRIPTION
**Pull request by nugaon**
export remaining functions of the library and add documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,64 @@
 
 With this package you can manipulate and interpret mantaray data via `MantarayNode` and `MantarayFork` abstractions.
 
+# Exported Functions and Classes
+
+You can import the followings directly from `mantaray-js`:
+
+* MantarayNode        # class for abstracting and manipulating Mantaray Node data
+* MantarayFork        # class abstracting and manipulating Mantaray Fork data
```
Comment (Cafe137) [inline README.md:10]: Inconsistent wording - please add missing "for"

File: README.md [hunk 2]
```diff
@@ -2,6 +2,64 @@
...
+const address5 = Utils.gen32Bytes()
+const path1 = new TextEncoder().encode('path1/valami/elso')
+const path2 = new TextEncoder().encode('path1/valami/masodik')
+const path3 = new TextEncoder().encode('path1/valami/masodik.ext')
+const path4 = new TextEncoder().encode('path1/valami')
+const path5 = new TextEncoder().encode('path2')
+node.addFork(path1, address1)
+node.addFork(path2, address2, { vmi: 'elso' }) // here 'vmi' is a key of metadata and 'elso' is its value
+node.addFork(path3, address3)
+node.addFork(path4, address4, { vmi: 'negy' })
```
Comment (Cafe137) [inline README.md:39]: I'd avoid adding Hungarian dummy words

**Review by Cafe137 — APPROVED** [review]
I had some notes regarding the wording, please review those. otherwise LGTM

File: README.md (hunk 2 above)
Comment (nugaon) [reply README.md:39]: Can you explain your stand of view?

**Pull request merged**